outputs/alertmanager.go

```diff
@@ -32,30 +32,28 @@ func newAlertmanagerPayload(falcopayload types.FalcoPayload, config *types.Confi
 			d, err := strconv.ParseInt(j.(string), 10, 64)
 			if err == nil {
 				var jj string
-				switch {
-				case d == 0:
+				if d == 0 {
+					if falcopayload.Priority < types.Warning {
+						falcopayload.Priority = types.Warning
+					}
```

Comment:
I don't understand why you keep this block if we can set thresholds

Reply:
To match the previous behavior that apply a Warning priority even if the `d` value is 0.
As the threshold works on `>` operator, a threshold at 0 will never be triggered.

Reply:
ok